tests/acceptance/bootstrap/SharingNgContext.php

```diff
@@ -508,6 +508,24 @@ public function userSendsTheFollowingSpaceShareInvitationToFederatedUserUsingPer
 		);
 	}
 
+	/**
+	 * @Given user :user has updated the last resource share with:
```

Comment:
this doesn't fall under you PR but could you look over this step too? It seems odd to me that different approaches have been used to do the same thing.
 `with the following` this also not seem good
```
    When user "Alice" updates the last resource share with the following using the Graph API:
```

Reply:
As we are checking the share update activities and there was no step to update a resource's share, that's why this step was created.
Can you elaborate more on what is confusing?